package.json

```diff
@@ -36,13 +36,13 @@
   },
   "dependencies": {
     "ramda": "^0.26.1",
-    "react": "^16.5.0",
-    "react-dom": "^16.5.0",
+    "react": "16.8.4",
+    "react-dom": "16.8.4",
```

Comment:
It'd probably make more sense to just get rid of these and just have the peerDependencies, and switch the peer dependencies to be `^16.x`. When you link locally you'll have to manually install the exact version glo uses, which is more annoying, but what you're doing here is assuming "linking locally" is the normal way this package will normally be used, which is incorrect.